docs/content/integrations/dbt/reference.mdx

```diff
@@ -537,20 +537,37 @@ Note that Dagster allows the optional specification of a [`code_version`](/conce
 
 ## Loading dbt tests as asset checks
 
-<Note>
-  <strong>Dagster asset checks are currently an experimental feature. </strong>{" "}
-  To provide feedback, join our{" "}
-  <a href="https://github.com/dagster-io/dagster/discussions/16527">
-    GitHub discussion
-  </a>{" "}
-  to share your use case with Dagster asset checks and dbt.
-</Note>
+Dagster loads your dbt tests as [asset checks](/concepts/assets/asset-checks).
+
+### Indirect selection
+
+Dagster uses [dbt indirect selection](https://docs.getdbt.com/reference/global-configs/indirect-selection) to select dbt tests. By default, Dagster won't set `DBT_INDIRECT_SELECTION` so that the set of tests selected by Dagster is the same as the selected by dbt. When required, Dagster will override `DBT_INDIRECT_SELECTION` to `empty` in order to explicitly select dbt tests. Examples:
```

Comment:
```suggestion
Dagster uses [dbt indirect selection](https://docs.getdbt.com/reference/global-configs/indirect-selection) to select dbt tests. By default, Dagster won't set `DBT_INDIRECT_SELECTION` so that the set of tests selected by Dagster is the same as those selected by dbt. When required, Dagster will override `DBT_INDIRECT_SELECTION` to `empty` in order to explicitly select dbt tests. For example:
```